src/index.js

```diff
@@ -452,6 +458,8 @@ const initialize = async () => {
       revokeButton.disabled = false;
       transferTokenInput.disabled = false;
       transferFromButton.disabled = false;
+      watchNFTButtons.innerHTML = '';
```

Comment:
NIT: Could we enable the `Watch All NFTs` button if the contract address is passed in as a query string param